src/holdingpen/containers/DashboardPage.jsx

```diff
@@ -1,10 +1,65 @@
 import React, { Component } from 'react';
+import data from '../report-errors.json';
+import CategoryCard from '../components/CategoryCard';
+import ExceptionsTable from '../components/ExceptionsTable';
+import { Row, Col } from 'antd';
 
 class DashboardPage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      exceptions: this.mapData(data),
+      collections: this.calculateExceptionsMetadata('collection')
+    }
+  }
+
+  mapData = (data) => {
+    const rawData = [...data];
+    const exceptions = rawData.map( (exception, index) => {
+      const keys = Object.keys(exception)
+      return ({
+        "key": index,
+        "collection": exception[keys[0]],
+        "errorType": exception[keys[1]],
+        "error": exception[keys[2]],
+        "record": exception[keys[3]]
+      })
+    })
+    return exceptions
+  }
+
+  calculateExceptionsMetadata = (property) => {
+    const exceptions = [...this.mapData(data)]
+
+    return exceptions.reduce( (tally, exception) => {
+      const index = tally.findIndex( category => category.type === exception[property]);
+      if(index===-1){ 
+        const obj = {type: exception[property], quantity: 1};
+        tally.push(obj);
+      } else {
+        tally[index]['quantity']++;
```

Comment:
no access with `['myProp']` unless prop name is dynamic or has characters like `-`